bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/ChannelUID.java

```diff
@@ -149,4 +149,22 @@ public ThingUID getThingUID() {
         List<String> allSegments = getAllSegments();
         return new ThingUID(allSegments.subList(0, allSegments.size() - 1).toArray(new String[allSegments.size() - 1]));
     }
+
+    /**
+     * Returns the binding id.
+     * 
+     * @return binding id
+     */
+    public String getBindingId() {
```

Comment:
I am wondering why this is needed? This class extends `UID` and there already is a `public` method with the same name available. Or did I miss something?